admin/orders.php

```diff
@@ -107,6 +107,19 @@
 
 if (!empty($action) && $order_exists === true) {
     $order = new order($oID);
+    $module = false;
+    global $installedPlugins;
+    $moduleFinder = new ModuleFinder('payment', new Filesystem());
+    $modules_found = $moduleFinder->findFromFilesystem($installedPlugins);
+    $payment = ($order->info['payment_module_code'] ?? 'nothing') . '.php';
+    if (array_key_exists($payment, $modules_found) && file_exists(DIR_FS_CATALOG . $modules_found[$payment] . $payment)) {
+        $payment_module = DIR_FS_CATALOG . $modules_found[$payment] . $payment;
+        $languageLoader->loadModuleLanguageFile($payment, 'payment');
+        if (file_exists($payment_module)) {
```

Comment:
Am I correct that this `file_exists()` is redundant because it's already checked 3 lines earlier?

Reply:
yes.  you are correct.  i'm just the worst coder ever!

i will fix and force push.